src/modules/newList.js

```diff
@@ -1,12 +1,21 @@
-/* eslint-disable linebreak-style */
+/* eslint-disable no-plusplus */
+let indexNumber = 0;
 
 class NewList {
   constructor() {
     this.array = [];
   }
 
-  add(Obj) {
-    this.array.push = new Obj();
+  add(obj) {
+    obj.index = indexNumber;
+    this.array.push(obj);
+    indexNumber++;
```

Comment:
- Hey, I don't seem to be able to add a task, the task doesn't get displayed on the UI, please, fix this issue.

Reply:
### EDITING FUNCTIONALITY

- Hey, you're yet to add the editing functionality, please, kindly add this missing part of the project requirement.